ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/actions/DeleteItemAction.java

```diff
@@ -111,15 +148,101 @@ public void apply(Void arg) throws OperationException {
         };
     }
 
-    private Operation<Void> actionComplete() {
+    private Operation<Void> actionComplete(final List<ResourceBasedNode<?>> nodes) {
+        final ProjectDescriptor project = appContext.getCurrentProject().getRootProject();
         return new Operation<Void>() {
             @Override
             public void apply(Void arg) throws OperationException {
                 if (actionCompletedCallBack != null) {
                     actionCompletedCallBack.onSuccess(null);
                 }
+
+                Map<String, List<String>> atributes = project.getAttributes();
+                if (!(atributes.containsKey("vcs.provider.name") && atributes.get("vcs.provider.name").contains("git"))) {
```

Comment:
Strange. As I know project.getMixins().contains("git") should return true only if project is under git. Because it is dynamic mixin. It looks like bug. Let me ask about it to @vparfonov WDYT?